### DISCUSSION TITLE
TuringCourses/others/abstract_algebra/

### DISCUSSION THREAD

**Discussion by giscus** [ZJU-Turing/TuringCourses#151] [General]
# TuringCourses/others/abstract_algebra/

图灵班学习指南

https://zju-turing.github.io/TuringCourses/others/abstract_algebra/

**Comment by shadowash0215** [comment]
我选择的是于飞老师的课程，的确不很尽如人意，这样难度的课程几乎只依靠自己看书还是有点过于困难了。我在学期中相当可观的时间都花在了这门课程的学习和作业中，而且还经常因为不太做得出来而饱受折磨。Hungerford 有一份广为流传但相当不全的作业答案，其中少数有答案的题目所使用的方法也不像是初学者能够使用的，所以我的建议是多搜索、多讨论。Mathematics Stack Exchange 中有不少与作业题类似的问题，并且很可能有多种视角的回答帮助你理解；实在不会的题目我也会请教学长等等。我也将我这学期的作业题进行了留档 [2023 - 2024 秋冬抽代作业题](https://note.shad0wash.cc/math/AbstractAlgebra/HW_archive/)，其中星号标记的是期末考试的原题范围。如果你发现我的解答有误欢迎纠正，或者有别的思路也欢迎留言指出！

**Comment by Jimmy-Chern** [comment]
推荐artin的书。顺序和讲课基本一致，基本不会出现预修知识不足的问题，而且引入比较自然，例如用环同态的核引入理想，不是概念和定理证明的罗列，阅读体验佳。另外由于讲课并没有覆盖教材的所有材料（可能只覆盖讲授章节的百分之六七十），反而是教材环之后的章节用到了一些范畴论的语言，影响阅读体验。另外artin的书有中译本（郭晋云），没有遇到因为翻译质量影响阅读的情况。
